pkg/kubelet/cm/container_manager.go

```diff
@@ -25,7 +25,7 @@ type ContainerManager interface {
 	// Runs the container manager's housekeeping.
 	// - Ensures that the Docker daemon is in a container.
 	// - Creates the system container where all non-containerized processes run.
-	Start() error
+	Start(*api.Node) error
```

Comment:
question: rather than passing in the node object directly, should we instead have an interface that more closely aligns with the information you need?  For example, a `NodeCapacityProvider` or something?